saksbehandler/src/refusjon/oversikt/Oversikt.tsx

```diff
@@ -33,14 +34,28 @@ const Oversikt: FunctionComponent = () => {
                 tableHeader={<SaksbehanderTableHeader />}
                 tableBody={<TabellBodySaksbehandler refusjoner={refusjonerPage.refusjoner} />}
             />
+            <div className={clsPagination.className}>
```

Comment:
Nå er jeg kanskje litt rusten på BEM, men er vi ikke allerede i context at `oversikt`-block'en? Blir det litt feil å ha en block inni en block? 🤔

Reply:
Tenker ikke at det gjør ikke noe at et "bem-hierarki" eksisterer inni et annet jeg 🤷

Men ➕ for css modules, det er nok flott å ta en oppfølgings-PR på tenker jeg!